src/Actions/Media/UpdateMedia.php

```diff
@@ -31,4 +41,20 @@ public function performAction(): Model
 
         return $media->withoutRelations();
     }
+
+    protected function validateData(): void
+    {
+        parent::validateData();
+
```

Comment:
Also validate if given collection exists for given model, and check for readonly